utils/lint_test_cases.sh

```diff
@@ -1,28 +1,7 @@
 #! /bin/bash
 
-for f in `find ./test-cases/ -type f \\( -iname "*.yaml" ! -iname "*multi-dc.yaml" ! -iname *multiDC*.yaml ! -iname *multiple-dc*.yaml ! -iname *rolling* ! -iregex .*docker.* ! -iregex .*artifacts.* !  -iregex .*private-repo.* ! -iregex .*ics/long.* ! -iregex .*scylla-operator/.* ! -iregex .*gce.* ! -iregex .*jepsen.* \\)` ; do
-    echo "---- linting: $f -----"
-    RES=$( script --flush --quiet --return /tmp/test-case.txt --command "SCT_SCYLLA_VERSION=4.0.0 python3 ./sct.py conf $f" )
-    if [[ "$?" == "1" ]]; then
-        cat /tmp/test-case.txt
-        exit 1;
-    fi
-done
+SCT_SCYLLA_VERSION=4.2.0 ./sct.py lint-yamls -i '.yaml' -e 'multi-dc.yaml,multiDC,multidc,multiple-dc,rolling,docker,artifacts,private-repo,ics/long,scylla-operator,gce,jepsen'
 
-for f in `find ./test-cases/ -type f \\( -iname *multi-dc.yaml -or -iname *multiDC*.yaml -or -iname *multiple-dc*.yaml -or -iname *rolling* -or -iregex .*artifacts.*yaml -or -iregex .*private-repo.*\.yaml -or -iregex .*gce.* -or -iregex .*jepsen.* \\) ! -name docker.yaml`; do
-    echo "---- linting: $f -----"
-    RES=$( script --flush --quiet --return /tmp/test-case.txt --command "SCT_SCYLLA_REPO=http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2019.1.repo python3 ./sct.py conf --backend gce $f" )
-    if [[ "$?" == "1" ]]; then
-        cat /tmp/test-case.txt
-        exit 1;
-    fi
-done
+SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2019.1.repo' ./sct.py lint-yamls -b gce -i 'multi-dc.yaml,multiDC,multidc,multiple-dc,rolling,artifacts,private-repo,gce,jepsen' -e 'docker'
```

Comment:
Make this one be newer too. 
For example, 2021.1.1 has already been released.

Reply:
done